packages/react-charting/src/components/DonutChart/DonutChartUnitTests.test.ts

```diff
@@ -0,0 +1,69 @@
+import * as utils from '../../utilities/utilities';
+
+describe('Unit test to convert data to localized string', () => {
+  test('Should return undefined when data provided is undefined', () => {
+    expect(utils.convertToLocaleString(undefined)).toBeUndefined();
+  });
+
```

Comment:
remove this file and move the md file under docs/TestPlans/Utilities.

Add a reference to the testplan file in the test.ts file as well

Reply:
done